src/Animation.re

```diff
@@ -55,11 +55,13 @@ let slideHorizontal = {
 
 let fadeVertical = {
   func:
-    Animated.Timing.animate(
-      ~duration=350.0,
-      ~easing=t => Js.Math.pow_float(~base=t, ~exp=5.0),
-      ~useNativeDriver=true,
-      (),
+    Animated.(
+      Timing.animate(
+        ~duration=350.0,
+        ~easing=Easing.out(Easing.poly(5.0)),
```

Comment:
Note this is actually even closer to the original animation as previous implementation was missing `Easing.out` call.